installer/install.iss

```diff
@@ -1048,14 +1048,27 @@ end;
 
 function EnableSymlinksByDefault():Boolean;
 var
+    SymlinksForRegularUsers:Cardinal;
+    Root:Integer;
     ResultCode:Integer;
 begin
-    if IsOriginalUserAdmin then begin
-        Log('Symbolic link permission detection failed: running as admin');
-        Result:=False;
```

Comment:
How about keeping the indentation intact by prefixing this code with the new code? You can use `Exit;` to return from the function early. Something like this:

```pascal
if [... detect developer mode ...] then begin
    [... log it ...]
    Result:=True;
    Exit;
end;
```

That will avoid the extra indentation, and will make reviewing the code easier (and as a consequence, it will make it harder for bugs to hide).

Reply:
Fixed.